src/Components/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const Button = ({onClick,children,className,value,onChange,type}) => {
+    return (
+
+        <button
+
+            onClick={onClick}
+            className={className}
+            value={value}
+            onChange={onChange}
+            type={type}
+        >
+            {children}
+        </button>
+    );
+};
```

Comment:
Dit component is nu een doorgeefluik. Alle properties die binnenkomen worden direct doorgezet op één component. Je kan dus net zo goed direct het html button element gebruiken